composer/metrics/nlp.py

```diff
@@ -451,3 +462,118 @@ def compute(self):
         assert isinstance(self.correct, Tensor)
         assert isinstance(self.total, Tensor)
         return self.correct.float() / self.total
+
+
+class InContextLearningExpectedCalibrationError(InContextLearningMetric):
```

Comment:
please add a test for this class. it can just be: 1) instantiate class 2) add some numbers to buckets manually 3) call compute and assert the correct number is gotten.